src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatService.java

```diff
@@ -73,14 +79,34 @@ public static TomcatService forFileSystem(Path docBase) {
         return TomcatServiceBuilder.forFileSystem(docBase).build();
     }
 
+    /**
+     * Creates a new {@link TomcatService} with existing web application instance.
+     * If web application is not configured properly, generated service may responds with 503 error
+     */
+    public static TomcatService forConnector(String hostname, Connector connector) {
+        requireNonNull(hostname, "hostname");
+        requireNonNull(connector, "connector");
+
+        return new TomcatService(new TomcatServiceInvocationHandler(hostname, connector));
```

Comment:
It seems slightly inconsistent with how managed tomcat service is created. We could probably pass hostname and connector to the constructor and let the constructor instantiate TomcatServiceInvocationHandler?